src/language-js/printer-estree.js

```diff
@@ -4982,7 +4982,12 @@ function printMemberChain(path, options, print) {
   // If the last group is a function it's okay to inline if it fits.
   if (
     hasComment ||
-    callExpressions.length >= 3 ||
+    (callExpressionCount >= 3 &&
```

Comment:
Where is callExpressionCount defined?

Reply:
Oops, looks like that didn’t get updated when I merged the latest changes in.